cmd/telemetrygen/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.17 AS build
+
+WORKDIR /src
+ADD . /src
+
+RUN CGO_ENABLED=0 go build -o /telemetrygen
+RUN ls
```

Comment:
nit: is the `ls` command from debugging or did you want to see the results of the build while the dockerfile is building?